.github/CHANGELOG.md

```diff
@@ -10,13 +10,52 @@
   [_Simple factorization of unitary transformations_](https://doi.org/10.1103/PhysRevA.97.022328).
   [(#665)](https://github.com/XanaduAI/strawberryfields/pull/665)
 
+  ```python
+  import numpy as np
+
+  import strawberryfields as sf
+  from strawberryfields import ops
+
+  U = np.array([[-0.39302099+0.28732291j,  0.83734522+0.24866248j],
+                [ 0.00769051+0.87345344j, -0.3847068 +0.29836325j]])
+
+  prog = sf.Program(2)
+
+  with prog.context as q:
+      ops.Interferometer(U, mesh="sun_compact") | q
+  ```
+
 * A `Device.certificate` method is added which returns the hardware device certificate.
   [(#679)](https://github.com/XanaduAI/strawberryfields/pull/679)
 
+  ```pycon
+  >>> import strawberryfields as sf
+  >>> eng = sf.RemoteEngine("X8")
+  >>> print(eng.device.certificate)
+  {'target': 'X8_01' ... }
+  ```
+
 * Setting `shots=None` in the engine or program run options will not execute any measurements
   applied on the circuit.
   [(#682)](https://github.com/XanaduAI/strawberryfields/pull/682)
 
+  ```python
+  import strawberryfields as sf
+  from strawberryfields import ops
+
+  prog = sf.Program(1)
+  eng = sf.Engine("gaussian")
+
+  with prog.context as q:
+      ops.Sgate(0.5) | q[0]
+      ops.MeasureFock() | q
+
+  results = eng.run(prog, shots=None)
+
+  print(results.samples)
+  # will output an empty list []
+  ```
```

Comment:
Here it would be cool to explicitly show that even if no samples are retuned the state is available
```suggestion
  print(results.samples)
  # will output an empty list []
  
  # the resulting Gaussian state is accesible 
  # via its vector of means and covariance matrix
  print(results.state.means())
  print(results.state.cov())
  ```
```